src/services/lnd/index.ts

```diff
@@ -316,3 +421,10 @@ const lookupPaymentByPubkeyAndHash = async ({
     return new UnknownLightningServiceError(err)
   }
 }
+
+const parseLndErrorDetails = (err) =>
```

Comment:
note: this was probably a hacky script I came along with? maybe this could be improve. or at least add a FIXME to it.

Reply:
This is how I would've done it too 🙈